themes/frontierline/content-views/content-mini.php

```diff
@@ -2,17 +2,23 @@
 /**
  * Display a post with just the thumbnail, title, and date.
  */
+$post_classes = 'post-mini';
+if (get_theme_mod('frontierline_post_thumbnail') !== '1') {
+  $post_classes .= ' post-thumb-disabled';
+}
 ?>
 
-<div class="post-mini">
+<div class="<?php echo $post_classes?>">
   <a class="entry-link" href="<?php the_permalink(); ?>">
-  <?php if (has_post_thumbnail()) :
-    $thumb_id = get_post_thumbnail_id();
-    $thumb_url = wp_get_attachment_image_src($thumb_id, 'post-thumbnail', true);
-  ?>
-    <img class="post-image" width="300" height="165" alt="" src="<?php echo get_template_directory_uri(); ?>/img/place-thumb.png" data-src="<?php echo $thumb_url[0]; ?>">
-  <?php else : ?>
-    <img class="post-image image-fallback color-<?php echo rand(1, 6); ?>" width="300" height="165" alt="" src="<?php echo get_template_directory_uri(); ?>/img/place-thumb.png" data-src="<?php echo get_template_directory_uri(); ?>/img/fallbacks/pattern-<?php echo rand(1, 6); ?>.png">
+  <?php if (get_theme_mod('frontierline_post_thumbnail') === '1') : ?>
```

Comment:
You should probably specify the default value in all [`get_theme_mod`](https://codex.wordpress.org/Function_Reference/get_theme_mod) usages. If I remember WordPress behaviour correctly, the setting will be null until you set it for the first time.

Reply:
It worked fine when I tested it but I think it would be safest for me to re-factor this as a "hide post thumbnails" checkbox so I can rely on `1` being a clear statement of intent.

+1 for more robust code. Thank you :)